php/deepgram_language_detection/README.md

```diff
@@ -0,0 +1,61 @@
+# deepgramLanguageDetection()
+
+A PHP Cloud Function for detecting language from audio using [Deepgram](https://deepgram.com/)
+
+_Example function payload:_
+
+```json
+{
+    "fileUrl": "https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Example successful function response:_
+
+```json
+{
+    "success": true,
+    "deepgramData": {}
+}
+```
+
+_Example error function response:_
+
+```json
+{
+    "success": false,
+    "message": "Please provide a valid file URL."
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Let's remove references to "Environment" since those are in the older runtime image:

```suggestion
## 📝 Variables

List of variables used by this cloud function:
```